scene/main/node.cpp

```diff
@@ -2091,74 +2092,6 @@ Node *Node::duplicate_from_editor(Map<const Node *, Node *> &r_duplimap) const {
 }
 #endif
 
-void Node::_duplicate_and_reown(Node *p_new_parent, const Map<Node *, Node *> &p_reown_map) const {
-
-	if (get_owner() != get_parent()->get_owner())
-		return;
-
-	Node *node = NULL;
-
-	if (get_filename() != "") {
```

Comment:
It took me a while to compare the old and the new logic to see if we're doing exactly the same thing. I think here we have a behaviour change though, as this condition is less strict than the one in `_duplicate()` (which checks for `(p_flags & DUPLICATE_USE_INSTANCING) && get_filename() != String()`, but we're not setting that flag).

Reply:
But then `_duplicate` also does a lot more stuff in case this is true (all the `instanced` logic).